services/ui-src/src/utils/api/requestMethods/report.ts

```diff
@@ -38,6 +38,18 @@ export async function putReport(report: Report) {
   );
 }
 
+export async function submitReport(report: Report) {
+  const requestHeaders = await getRequestHeaders();
+  const options = {
+    headers: { ...requestHeaders },
+    body: { ...report },
+  };
+  return await apiLib.post(
+    `/reports/submit/${report.type}/${report.state}/${report.id}`,
```

Comment:
curious why we're doing plural for report here.
`reports` this isn't the URL we see in the browser right?

Reply:
this is the api url, not the ui, and is defined in the serverless file